### PR TITLE
[RFR] improving test coverage

### PR DESCRIPTION
**Pull request by Rabinzon**
issue https://github.com/mbrandau/beautiful-logs/issues/1

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -1,22 +1,49 @@
...
+        it(`should be called ${level}`, generateTest(level));
       });
     });
+
+    describe('.lineEmpty()', function() {
+      it('should be called once', function() {
+          const logger = init();
+          sinon.spy(logger, 'emptyLine');
+          logger.emptyLine('trace', 'Test message!');
+          expect(logger.emptyLine.calledOnce).to.be.true;
```
Comment (mbrandau) [inline test/index.js]: It doesn't make any sense to spy on the .empyLine() function. `sinon.spy(console,'log')` (if that is the correct way) is the better solution.

File: test/index.js [hunk 2]
```diff
@@ -1,22 +1,49 @@
...
+      levels.forEach(level => {
+        it(`should be called ${level}`, generateTest(level));
       });
     });
+
+    describe('.lineEmpty()', function() {
+      it('should be called once', function() {
+          const logger = init();
+          sinon.spy(logger, 'emptyLine');
+          logger.emptyLine('trace', 'Test message!');
```
Comment (mbrandau) [inline test/index.js]: .emptyLine() should be called without any arguments

File: test/index.js [hunk 3]
```diff
@@ -1,22 +1,49 @@
...
+          expect(logger.emptyLine.calledOnce).to.be.true;
+      })
+    })
+
+    describe('.line()', function() {
+      it('should be called once', function() {
+        const logger = init();
+        sinon.spy(logger, 'line');
+        logger.line();
+        expect(logger.line.calledOnce).to.be.true;
```
Comment (mbrandau) [inline test/index.js]: Same problem as in line 36

**Comment by Rabinzon** [comment]
@mbrandau fixed :)

**Comment by mbrandau** [comment]
Looking good! Thanks! :+1:

**Pull request merged**